test/plugins.spec.js

```diff
@@ -114,39 +114,56 @@ test('add options, merging correctly', async () => {
   });
 });
 
-test('override duplicates in smartContract.addContracts with the new address', async () => {
-  const exampleAbiItem = {
-    constant: true,
-    inputs: [],
-    name: 'test',
-    outputs: [
-      {
-        name: '',
-        type: 'address'
-      }
-    ],
-    payable: false,
-    stateMutability: 'view',
-    type: 'function'
+test('plugins using addContracts property will override the previous one in sequential order; user-supplied config will override all', async () => {
```

Comment:
you have good comments in the body of the test so this name can be shorter, just something like 'plugin override rules'